.github/workflows/upstream.yml

```diff
@@ -63,69 +63,21 @@ jobs:
 
       - name: Run tests
         id: run_tests
+        env:
+          XTRATESTARGS: "--report-log output-log.jsonl"
         run: source continuous_integration/scripts/run_tests.sh
 
+      - name: Open or update issue on failure
+        if: |
+          failure()
+          && github.event_name != 'pull_request'
+          && github.repository == 'dask/dask'
+          && steps.run_tests.outcome == 'failure'
+        uses: xarray-contrib/issue-from-pytest-log@1.1
```

Comment:
Should this be

```suggestion
        uses: xarray-contrib/issue-from-pytest-log@v1.1
```

Reply:
Ah, you're right, thanks for catching that!